### PR TITLE
Use links in OEP-26 that stay within the published docs

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

File: oeps/oep-0026-arch-realtime-events.rst [hunk 1]
```diff
@@ -120,14 +120,14 @@ In order to enable a plug-n-play ecosystem with various external services (adap
 xAPI Integration
 ~~~~~~~~~~~~~~~~
 
-For details on integrating with xAPI, please see the `xAPI Real-time Events`_ design document.
+For details on integrating with xAPI, please see the :ref:`xapi_realtime_events` design document.
 
 .. _xAPI Real-time Events: https://github.com/edx/open-edx-proposals/blob/master/oeps/oep-0026/xapi-realtime-events.rst
```
Comment (nasthagiri) [inline oeps/oep-0026-arch-realtime-events.rst:125]: Can we delete this line now?

File: oeps/oep-0026-arch-realtime-events.rst [hunk 2]
```diff
@@ -120,14 +120,14 @@ In order to enable a plug-n-play ecosystem with various external services (adap
...
 
 Caliper Integration
 ~~~~~~~~~~~~~~~~~~~
 
-For details on integrating with Caliper, please see the `Caliper Real-time Events`_ design document.
+For details on integrating with Caliper, please see the :ref:`caliper_realtime_events` design document.
 
 .. _Caliper Real-time Events: https://github.com/edx/open-edx-proposals/blob/master/oeps/oep-0026/caliper-realtime-events.rst
```
Comment (nasthagiri) [inline oeps/oep-0026-arch-realtime-events.rst:132]: And delete this one as well?

**Review by nasthagiri — APPROVED** [review]
Is it as simple as that? Thank YOU!!

File: oeps/oep-0026-arch-realtime-events.rst (hunk 1 above)
Comment (nedbat) [reply oeps/oep-0026-arch-realtime-events.rst:125]: Oops! yes. on it.

Comment (nedbat) [reply oeps/oep-0026-arch-realtime-events.rst]: Done

**Review by feanil — APPROVED** [review]

**Pull request merged**